source/common/grpc/common.h

```diff
@@ -25,6 +25,34 @@ class Exception : public EnvoyException {
 
 class Common {
 public:
+  enum GrpcStatus {
```

Comment:
It would be helpful if this could be moved to `include/envoy/grpc/status.h`, since I would like to use this in interface files for the bidi client I'm working on.